x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -24,210 +24,252 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         });
-        it('should delete index', async () => {
-          await pageObjects.svlSearchIndexDetailPage.expectDeleteIndexButtonExistsInMoreOptions();
-          await pageObjects.svlSearchIndexDetailPage.clickDeleteIndexButton();
-          await pageObjects.svlSearchIndexDetailPage.clickConfirmingDeleteIndex();
+        after(async () => {
+          await esDeleteAllIndices(indexName);
+        });
+        describe('manage index action', () => {
+          beforeEach(async () => {
+            await pageObjects.indexManagement.manageIndex(indexName);
+            await pageObjects.indexManagement.manageIndexContextMenuExists();
+          });
+          it('navigates to overview tab', async () => {
+            await pageObjects.indexManagement.changeManageIndexTab('showOverviewIndexMenuButton');
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('data');
+          });
+
+          it('navigates to settings tab', async () => {
+            await pageObjects.indexManagement.changeManageIndexTab('showSettingsIndexMenuButton');
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('settings');
+          });
+          it('navigates to mappings tab', async () => {
+            await pageObjects.indexManagement.changeManageIndexTab('showMappingsIndexMenuButton');
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectUrlShouldChangeTo('mappings');
+          });
+        });
+        describe('can view search index details', function () {
+          it('renders search index details with no documents', async () => {
+            await pageObjects.svlSearchIndexDetailPage.openIndicesDetailFromIndexManagementIndicesListTable(
+              0
+            );
+            await pageObjects.svlSearchIndexDetailPage.expectIndexDetailPageHeader();
+            await pageObjects.svlSearchIndexDetailPage.expectSearchIndexDetailsTabsExists();
+            await pageObjects.svlSearchIndexDetailPage.expectAPIReferenceDocLinkExists();
+          });
         });
       });
     });
-    describe('index management index details', () => {
+
+    describe('viewer', function () {
       before(async () => {
-        await es.indices.create({ index: indexName });
-        await security.testUser.setRoles(['index_management_user']);
-      });
-      beforeEach(async () => {
-        await pageObjects.common.navigateToApp('indexManagement');
-        // Navigate to the indices tab
-        await pageObjects.indexManagement.changeTabs('indicesTab');
-        await pageObjects.header.waitUntilLoadingHasFinished();
+        await esDeleteAllIndices(indexName);
+        await es.index({
```

Comment:
**not for this PR**, but I think we should create all the indices and ingest docs in the very top level `before()` then just use different indices to check has data, has mapping etc. This should also help with flakiness. I will add an issue to the backlog for this.